browser/src/layer/tile/ImpressTileLayer.js

```diff
@@ -61,6 +61,12 @@ L.ImpressTileLayer = L.CanvasTileLayer.extend({
 			else
 				targetElement.classList.remove('selected');
```

Comment:
Unrelated but I noticed: an ugly state handler - we could do it automatically with state update for an uno command. Do we send notes command state from core? we should.

Reply:
Yeah but i think that's the way we use for toggle buttons. But still, we may consider a better approach as you say. I think it'd take some time so i am passing it for now :)